src/styles/main.scss

```diff
@@ -1,3 +1,53 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+$days: ("mon":0, "tue":1, "wed":2, "thu":3, "fri":4, "sat":5, "sun":6);
+$cell_width:100px;
+$gap:1px;
+$month_length:(28,29,30,31);
+@mixin calendar_block_cell {
+  width: 100px;
+  height: 100px;
+  background-color: #eee;
+  border: 1px solid black;
+}
+
+.calendar {
```

Comment:
It is recommended to place each BEM block in its own file. Therefore, kindly move the calendar with all its elements to the calendar.scss file.